index.js

```diff
@@ -76,6 +76,11 @@ const run = async () => {
         continue
       }
 
+      if (aggregate && issues.find(x => x.title.startsWith(issueTitlePrefix) && Date.parse(x.created_at) > Date.parse(item.isoDate))) {
```

Comment:
I agree that we need to have special handling for `aggregate` here, and I think that date check makes a ton of sense.

Maybe _in addition_ to that, we could use the `isoDate` of the first aggregated item as the label (instead of `now`)?

Reply:
I've thought about that, but I wasn't sure if `rss-parser` actually guarrantees the first item is the latest. And we still wouldn't be able to just compare the title for equality, unless we'd be willing to accept one more duplicate after switching from the `now` version to the `isoDate` version.

Reply:
Good points!